huey/contrib/djhuey/__init__.py

```diff
@@ -105,10 +105,10 @@ def close_db(fn):
     @wraps(fn)
     def inner(*args, **kwargs):
         try:
+            close_db_connections()
```

Comment:
Why close the connections here? The purpose of this decorator is to close the connections after executing the callback.

Reply:
This is the same behaviour as django. Before every request old/broken connections get closed and after every request old/broken connections get closed.
From https://github.com/django/django/blob/master/django/db/__init__.py
```python
signals.request_started.connect(close_old_connections)
signals.request_finished.connect(close_old_connections)
```
I'm not 100% sure but my guess would be so that it closes broken/old connections from the pool before it starts processing a request.
But I can change it to only close old connections after the job.

Reply:
It seems like a code-smell to me, so let's remove it from the huey implementation. If you don't mind making that change, I'd be happy to merge...maybe you could, while you're at it, rebase into a single clean patch?

Also, what is the minimum Django version that supports this API?